yt_dlp/extractor/npo.py

```diff
@@ -257,7 +257,8 @@ def _real_extract(self, url):
             format_urls.add(stream_url)
             if stream.get('protection') is not None or stream.get('keySystemOptions') is not None:
                 drm = True
-                continue
+                if not self.get_param('allow_unplayable_formats'):
+                    continue
```

Comment:
I don't agree with this. I don't want this merged. And I definitely don't want this merged in a PR with my name on it
```suggestion
                continue
```